### DISCUSSION TITLE
clever_u throwing session related error on Keras Classifier

### DISCUSSION THREAD

**Discussion by ghost** [Trusted-AI/adversarial-robustness-toolbox#1263] [Q&A] [answered]
Greetings,
I was trying the untargeted clever score on a Keras model. I am getting the following error:

> FailedPreconditionError: Could not find variable dense_10/kernel. This could mean that the variable has been deleted. In TF1, it can also mean the variable is uninitialized. Debug info: container=localhost, status=Not found: Resource localhost/dense_10/kernel/N10tensorflow3VarE does not exist.
> 	 [[{{node dense_10/MatMul/ReadVariableOp}}]]

I would appreciate any help.
The code, model and the data is attached.
Thank you,
Melike
[clever.zip](https://github.com/Trusted-AI/adversarial-robustness-toolbox/files/6967104/clever.zip)

**Comment by beat-buesser** [comment, marked as answer]
Hi @melikedem Are you still observing this error?

**Reply by ghost** [reply]
Hello,
Yes, I would be so happy if you could guide me through the reason of the error.
Thank you :)

**Reply by beat-buesser** [reply]
Hi @melikedem I have opened your notebook and run all cells successfully and the output of `clever_u` is `0.1493562299346208`. Is there an additional step to reproduce the error?

**Comment by ghost** [comment]
Hello again,
No, there is no additional step. Since then I tried the same code in 2 other machines and it works. However I still get the same error on my machine I think we can just ignore it and assume that it is specific to my computer.
I am so sorry for bothering you.
Thank you so much for your time,
Melike

**Reply by beat-buesser** [reply]
@melikedem No worries at all. That's actually interesting, which operating system are you using on your computer? Are there any other major differences between you computer and the other 2 machines?

**Reply by ghost** [reply]
Hello,
My operating system is macOS (Big Sur 11.1 specifically). I tried one Linux machine and another Mac. I have been looking into it but I failed to find any significant differences. I checked the versions of the installed libraries but it did not help.
Unfortunately I cannot provide something useful.
Thank you anyways :)
Best,
Melike
-----EDIT
I just found the problem. With the inspiration of this https://stackoverflow.com/a/60947887 I changed my library import to be

> from tensorflow.keras.models import load_model

instead of

> from keras.models import load_model

Sorry for taking your time for something completely irrelevant to ART.
Thank you so much for your kind approach and help,
Best regards,
Melike
